account/forms/organization.py

```diff
@@ -0,0 +1,41 @@
+from django import forms
+from django.utils.translation import gettext_lazy as _
+from tools.models import Organization, OrganizationMember
+from tools.forms.settings import BLANK_CHOICE
+
+
+class OrganizationListForm(forms.Form):
+    """
+    Creates a dropdown list of Organizations of a particular member.
+    """
+
+    error_messages = {
+        "required": _("Organization is required."),
+    }
+
+    def __init__(self, user, exclude_organization=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.exclude_organization = exclude_organization
+        self.populate_dropdown_list(user)
+
+    def populate_dropdown_list(self, user):
+        organizations = []
+
+        members = OrganizationMember.objects.filter(user=user)
+        if members.exists():
```

Comment:
This can be removed; if there are no members then you would just iterate over an empty sequence